internal/services/search/parse/search_service.go

```diff
@@ -1,69 +0,0 @@
-package parse
```

Comment:
we also need to remove the entry from `resourceids.go` : https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/search/resourceids.go#L3

Reply:
Done!